packages/core/src/execution/dispatch/executeSendETH.ts

```diff
@@ -0,0 +1,38 @@
+import type { PopulatedTransaction } from "ethers";
+
+import { ExecutionContext } from "types/deployment";
+import { SentETH } from "types/executionGraph";
+import { VertexVisitResult } from "types/graph";
+
+import { resolveFrom, toAddress } from "./utils";
+
+export async function executeSendETH(
+  { address, value }: SentETH,
+  resultAccumulator: Map<number, VertexVisitResult | null>,
+  { services, options }: ExecutionContext
+): Promise<VertexVisitResult> {
+  const resolve = resolveFrom(resultAccumulator);
+
+  const to = toAddress(resolve(address));
+
+  let txHash: string;
+  try {
+    const tx: PopulatedTransaction = { to, value };
+
+    txHash = await services.contracts.sendTx(tx, options);
+  } catch (err) {
+    return {
+      _kind: "failure",
+      failure: err as any,
+    };
+  }
+
+  await services.transactions.wait(txHash);
```

Comment:
I have been meaning to ask about this. The `sendTx` call is within the catch by the `wait` call is not. Are there any cases where `wait` can blow up?

I was thinking about this when looking over the execute call code. I didn't make a change, but wanted to come back to it.

Reply:
Looking into the ethers code, it looks like it can throw on a timeout exceeded or if the given tx hash is replaced on-chain. I just copied this code from `executeContractCall`. I can update all of these in another PR

Reply:
Agreed. It was looking at the `executeContractCall` last week that got me wondering. I will open a linear ticket for this.